python/py-neovim/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           python 1.0
 PortGroup           github 1.0
 
-github.setup        neovim pynvim 0.4.2
+github.setup        neovim pynvim 0.4.3
```

Comment:
Why not update to [latest 0.4.4](https://github.com/neovim/neovim/releases/tag/v0.4.4)?